spec/sections/standard.adoc

```diff
@@ -176,14 +177,14 @@ input file size for scaling memory requirement of job.
 The Interface object is the primary member that describes the command arguments, environment variables,
 mounts, settings, inputs and outputs defined for a job.
 
-* The Interface object MAY have a member with the name `cmd`. The member's value MUST be a string specifying the
+* The Interface object MAY have a member with the name `command`. The member's value MUST be a string specifying the
```

Comment:
It is definitely true and you are correct we do expect them to set an ENTRYPOINT. The Docker CMD always is additional arguments to the ENTRYPOINT command. It is just usually the default ENTRYPOINT is /bin/sh or the like which results in CMD appearing to be the full executable AND args.